tests/src/test/scala/org/apache/openwhisk/core/limits/ConcurrencyTests.scala

```diff
@@ -101,55 +101,6 @@ class ConcurrencyTests extends TestHelpers with WskTestHelpers with WskActorSyst
       }
   }
 
-  //This tests generates the same load against the same action as previous test, BUT with concurrency set to 1
-  it should "execute activations sequentially when concurrency = 1 " in withAssetCleaner(wskprops) {
-    assume(Option(WhiskProperties.getProperty("whisk.action.concurrency", "False")).exists(_.toBoolean))
-
-    (wp, assetHelper) =>
-      val name = "TestNonConcurrentAction"
-      assetHelper.withCleaner(wsk.action, name, confirmDelete = true) {
-        val actionName = TestUtils.getTestActionFilename("concurrent.js")
-        (action, _) =>
-          //disable log collection since concurrent activation requires specialized log processing
-          // (at action runtime and using specialized LogStore)
-          action.create(name, Some(actionName), logsize = Some(0.bytes), concurrency = Some(1))
-      }
-      //warm the container (concurrent activations with no warmed container, will cause multiple containers to be used - so we force one to warm up)
-      val run = wsk.action.invoke(name, Map("warm" -> 1.toJson), blocking = true)
-      withActivation(wsk.activation, run) { response =>
-        val logs = response.logs.get
-        withClue(logs) { logs.size shouldBe 0 }
-
-        response.response.status shouldBe "success"
-        response.response.result shouldBe Some(JsObject("warm" -> 1.toJson))
-      }
-
-      //read configs to determine max concurrency support - currently based on single invoker and invokerUserMemory config
-      val busyThreshold =
-        (loadConfigOrThrow[ContainerPoolConfig](ConfigKeys.containerPool).userMemory / MemoryLimit.STD_MEMORY).toInt
-
-      //run maximum allowed concurrent actions via Futures
-      val requestCount = busyThreshold
-      println(s"executing $requestCount activations")
-      val runs = (1 to requestCount).map { _ =>
-        Future {
-          //expect only 1 activation concurrently (within the 1 second timeout implemented in concurrent.js)
-          wsk.action.invoke(name, Map("requestCount" -> 1.toJson), blocking = true)
```

Comment:
This test case also assumes the sharding pool balancer is used.
It expects multiple containers to be created and each of them receives only 1 request each.
But with the new scheduler, one container could receive multiple activations.